http/handler.go

```diff
@@ -567,18 +567,41 @@ func respondStandby(core *vault.Core, w http.ResponseWriter, reqURL *url.URL) {
 	w.WriteHeader(307)
 }
 
+// getTokenFromReq parse headers of the incoming request to extract token if present
+// it accepts Authorization Bearer (RFC6750) and X-Vault-Token header
+func getTokenFromReq(r *http.Request) (string, error) {
+	if token := r.Header.Get(consts.AuthHeaderName); token != "" {
+		return token, nil
+	}
+	if v := r.Header.Get("Authorization"); v != "" {
+		// Reference for Authorization header format: https://tools.ietf.org/html/rfc7236#section-3
+
+		if !strings.HasPrefix(v, "Bearer") {
```

Comment:
RFC 6750 gives only one example of value `v`: `Bearer mF_9.B5f-4.1JqM`. I wonder if it would be safe to check for `"Bearer "` here.